contents/handbook/engineering/ee-setup.md

```diff
@@ -42,6 +42,7 @@ brew install libxml2 libxmlsec1 pkg-config
 - Run the frontend
   - `yarn build`
   - `yarn start` or click "▶️" next to `"start"` in the scripts section of package.json.
+  - Alternatively, you can also run `./bin/start-frontend` or `yarn start-http --host 0.0.0.0` if you're running into issues with `yarn start`
```

Comment:
I think we should just recommend using `yarn start-http --host 0.0.0.0` instead of `yarn start` since `./bin/start-frontend` nor `yarn start` will work with Docker setup

Reply:
how come that doesn't work with the docker setup?

Reply:
irrc `yarn start` on its own doesn't host the frontend distribution files on the default localhost. Out local server can't talk to Docker and vice versa unless we manually create a tunnel. That's why serving it on a universally accepted host like 0.0.0.0 will work. I actually think there's a new script called `start-docker:esbuild` that aliases the command you wanna run.